FogCreek/Glitch-Community
# Potent-Skink: Edit Project Page buttons based on viewport size

## Links
* Remix link: https://potent-skink.glitch.me/~community
* Issue-tracker link: https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/503

## GIF/Screenshots:
![button-contents](https://user-images.githubusercontent.com/519336/66430929-0ca06200-e9e9-11e9-96c8-2e9ebebdadb4.gif)

After Change / Before:
![image](https://user-images.githubusercontent.com/519336/66431198-8df7f480-e9e9-11e9-800c-a71657d7b116.png)

## Changes:
* Change content of buttons under the embed on Project Pages based on size of viewPort (breakpoint at mediumSmallViewport of 592px).  This helps the buttons look better on mobile.

Text changes:
Edit Project -> Edit
Join Team Project -> Join
Add to Collection -> Add
Remix Your Own / Remix This -> Remix

## How To Test:
* Log into https://potent-skink.glitch.me/~community
* Resize the viewport / view for mobile screen to see how the button text changes

## Feedback I'm Looking For:
* We should only have one CTA on a page, so I changed the Edit Button that appears below the project embed to a normal button, rather than a CTA one
